test/e2e/cypress/support/index.d.ts

```diff
@@ -309,6 +309,7 @@ declare global {
        * @param opts
        */
       registerOidc(opts: {
+        app: string
```

Comment:
This can only ever be `react` or `express` right? If so, this should also just be the enum. Then you also don't need the typecasting in the implementation.